pkgs/tools/backup/gamerbackup/default.nix

```diff
@@ -1,24 +1,28 @@
-{ lib, buildGoModule, fetchFromGitHub, lepton }:
+{ lib, makeWrapper, buildGoModule, fetchFromGitHub, lepton }:
 
 buildGoModule {
   pname = "gb-backup";
-  version = "unstable-2021-03-06";
+  version = "unstable-2021-04-07";
 
   src = fetchFromGitHub {
     owner = "leijurv";
     repo = "gb";
-    rev = "5a94e60148628fc7796d15c53d0ed87184322053";
-    sha256 = "07skhwnxvm6yngb2665gkh5qbiyp7hb7av8dkckzypmd4k8z93cm";
+    rev = "904813bf0bbce048af5795618d58c0b1953f9ff8";
+    sha256 = "111jrcv4x38sc19xha5q3pd2297s13qh1maa7sa1k09hgypvgsxf";
   };
 
   vendorSha256 = "0m2aa6p04b4fs7zncar1mlykc94pp527phv71cdsbx58jgsm1jnx";
 
-  buildInputs = [ lepton ];
+  nativeBuildInputs = [ makeWrapper lepton ];
```

Comment:
```suggestion
  nativeBuildInputs = [ makeWrapper ];
```

Reply:
lepton needs to be in the PATH for the test to work

Reply:
should be fixed now